lib/src/scheduler.rs

```diff
@@ -11,3 +11,51 @@ where
     fn set_processor(&mut self, processor: &T);
     fn schedule(&mut self) -> (i32, VecDeque<NodeIndex>);
 }
+
+#[derive(Clone, Default)]
+pub struct NodeLog {
+    pub core_id: usize,
+    pub node_id: i32,
+    pub start_time: i32,
+    pub end_time: i32,
```

Comment:
In real-time scheduling domain, `finish_time` is more common.

Reply:
Modified.